docs/site/translation/displaynames/locale-option-names-key.md

```diff
@@ -1,38 +1,321 @@
-| Private Use | Used for Private-Use options (x). |
+When displayed as a menu or in certain other contexts, the name of the _key_ for the option (such as _Calendar_) is split from the names of the different _values_ for that option (such as _Buddhist, Gregorian, Japanese_, etc.) That shorter version is called the core value.
 
-## Locale Option Value Names
-
-There are two kinds of option value names: the Long name and the Core name. The Long name includes the name of the Option while the Core name does not. Here are examples of what that can look like:
+So you will be asked to translate the long name (such as _Buddhist Calendar_), plus the name of each key (_Calendar_) and the name of the "core" value (_Buddhist_). This allows systems to show users of your language what all the locale options are.
 
 | Code | Name |
 | -- | -- |
 | `calendar` | Kalender |
 | `calendar-buddhist` | Buddhistischer Kalender |
 | `calendar-buddhist-core` | Buddhistischer |
 
-The core name is used in two ways: 
-- In locale names with Locale Option Pattern, like "English (Calendar: Buddhist)"
-- In menu listings or pull-downs, where the Option code is used as the header, such as:
+The core name is typically used in menu listings or pull-downs, where the key name is used as the header, such as:
     - **Calendar**
         - Buddhist
         - Chinese
         - Coptic
         - …
+     
+This avoids the repetition that you would see if the long name were used:
+    - **Calendar**
+        - Buddhist Calendar
+        - Chinese Calendar
+        - Coptic Calendar
+        - …
+
+The core name can aso be used as an alternate form of a full locale name, such as "English (UK, Calendar: Buddhist)".
+
+
+## Locale Option Names
+
+Here are examples of names of Options to be translated.
+
+| Key | Meaning   |
+|---|---|
+| Calendar | Calendar system (the European calendar is called "Gregorian"; others are the Chinese Lunar Calendar, and so on.) |
+| Collation | How text is sorted (where a language has different possible ways to sort). |
+| Currency | The default currency. (The value is any currency value, such as USD). |
+| Numbers | The numbering system in use, such as European (0,1,2), Arabic (٠, ١, ٢ ), Devanagari ( ०,  १,  २ ). <br /> - Usually these are just derived from the name of the script.<br /> - There are some special forms, such as " Simplified Chinese Financial Numerals " or " Full Width Digits ". |
+
+## Full List
+The following provides a full list of keys and values. You may see just a subset of the keys and/or their values, depending on your coverage level. (It does exclude the Transform options, because those are lesser-used.)
+
+Note that in many cases the values do not need to be translated, because the data is available elsewhere. For example, most numbering systems ar identified by their Script code. Those are indicated in the list below by all-uppercase Codes.
```

Comment:
```suggestion
Note that in many cases the values do not need to be translated, because the data is available elsewhere. For example, most numbering systems are identified by their Script code. Those are indicated in the list below by all-uppercase Codes.
```